Sources/Nuke/Internal/Graphics.swift

```diff
@@ -366,7 +383,22 @@ func makeThumbnail(data: Data, options: ImageRequest.ThumbnailOptions) -> Platfo
     guard let image = CGImageSourceCreateThumbnailAtIndex(source, 0, options as CFDictionary) else {
         return nil
     }
+    
+#if canImport(UIKit)
+    let orientation: UIImage.Orientation
```

Comment:
this is needed to make sure the returned image has the same orientation as the original. Verified via tests below with .right and .up orientation images